stable/phpmyadmin/README.md

```diff
@@ -0,0 +1,80 @@
+# PhpMyAdmin
+
+[Phpmyadmin](https://www.phpmyadmin.net/) is a free and open source administration tool for MySQL and MariaDB. As a portable web application written primarily in PHP, it has become one of the most popular MySQL administration tools, especially for web hosting services.
+
+## TL;DR;
+
+```console
+$ helm install stable/phpmyadmin
+```
+
+## Introduction
+
+This chart bootstraps a [Phpmyadmin](https://github.com/bitnami/bitnami-docker-phpmyadmin) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/phpmyadmin
+```
+
+The command deploys Phpmyadmin on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the Phpmyadmin chart and their default values.
+
+|              Parameter               |               Description                |                         Default                         |
+|--------------------------------------|------------------------------------------|---------------------------------------------------------|
+| `image.registry`                     | PhpMyAdmin image registry                 | `docker.io`                                             |
+| `image.repository`                   | PhpMyAdmin Image name                     | `bitnami/phpmyadmin`                                     |
+| `image.tag`                          | PhpMyAdmin Image tag                      | `{VERSION}`                                             |
+| `image.pullPolicy`                   | Image pull policy                        | `Always` if `imageTag` is `latest`, else `IfNotPresent` |
```

Comment:
The default value column should just say `IfNotPresent`

Reply:
done